modules/cluster-logging-troubleshooting-loki-entry-out-of-order-errors.adoc

```diff
@@ -41,7 +41,7 @@ If your Fluentd forwards a large block of messages to a Loki logging system that
 ----
 ,\nentry with timestamp 2021-08-18 05:58:55.061936 +0000 UTC ignored, reason: 'entry out of order' for stream:
 
-{fluentd_thread=\"flush_thread_0\", log_type=\"audit\"},\nentry with timestamp 2021-08-18 06:01:18.290229 +0000 UTC ignored, reason: 'entry out of order' for stream: {fluentd_thread="flush_thread_0", log_type="audit"}
+{collector_thread=\"flush_thread_0\", log_type=\"audit\"},\nentry with timestamp 2021-08-18 06:01:18.290229 +0000 UTC ignored, reason: 'entry out of order' for stream: {collector_thread="flush_thread_0", log_type="audit"}
```

Comment:
`fluentd_thread` is a label set by fluentd here, so you can't rename it to `collector_thread`